test/issue-generator.js

```diff
@@ -34,28 +35,54 @@ export default function createTestIssue(key, type, color, parentKey, parentType,
       }
     }
   };
+  return baseIssue;
```

Comment:
Since you're here, you may as well return the object directly and not even bother saving it to a variable.